packages/hooks/wallet/getNativeWallet.ts

```diff
@@ -1,10 +1,15 @@
 import { Secp256k1HdWallet } from "@cosmjs/amino";
+import { stringToPath } from "@cosmjs/amino/node_modules/@cosmjs/crypto/build/slip10";
```

Comment:
I had to do this. To make tsc stop complaining
Do you know of any other options?